src/DssSpell.t.sol

```diff
@@ -226,9 +226,9 @@ contract DssSpellTest is DSTest, DSMath {
         // Test for spell-specific parameters
         //
```

Comment:
I think lines L124-127 can be removed (as specific to prev spell)

Reply:
Done.